### PR TITLE
Fixing tags in helm charts

### PR DESCRIPTION
**Pull request by utka**
Currently all releases (daily, RCs) have tag 1.1.0. Instead I am updating current_tag to $CB_VERSION
Which is 1.1.X for RCs and master-2019XY-09-16 for daily.

### CODE REVIEW

File: release/gcb/helm_values.sh [hunk 1]
```diff
@@ -33,6 +33,10 @@ function fix_values_yaml() {
   # Update version string in yaml files.
   sed -i "s|hub: gcr.io/istio-release|hub: ${CB_DOCKER_HUB}|g" ./"istio-${CB_VERSION}"/install/kubernetes/helm/istio*/values.yaml
   sed -i "s|tag: .*-latest-daily|tag: ${CB_VERSION}|g"         ./"istio-${CB_VERSION}"/install/kubernetes/helm/istio*/values.yaml
+  current_tag=$(grep "appVersion" ./"istio-${CB_VERSION}"/install/kubernetes/helm/istio/Chart.yaml  | cut -d ' ' -f2)
+  if [ ${current_tag} != ${CB_VERSION} ]; then
+    find . -type f -exec sed -i "s/${current_tag}/${CB_VERSION}/g" {} \;
```
Comment (howardjohn) [inline release/gcb/helm_values.sh]: I don't think this is safe, we could easily have "1.1.0" or whatever in a chart (that shouldn't be replaced)

Comment (utka) [reply release/gcb/helm_values.sh]: Agree, not sure what will be safe way of doing it.
Current list of files I was experimenting with:
istio-init/values.yaml:8:  tag: 1.1.7 
istio-init/Chart.yaml:2:appVersion: 1.1.7 
istio-init/Chart.yaml:13:version: 1.1.7 
istio/requirements.yaml:3:    version: 1.1.7 
istio/requirements.yaml:6:    version: 1.1.7 
istio/requirements.yaml:9:    version: 1.1.7 
istio/requirements.yaml:12:    version: 1.1.7 
istio/requirements.yaml:15:    version: 1.1.7 
istio/requirements.yaml:18:    version: 1.1.7 
istio/requirements.yaml:21:    version: 1.1.7 
istio/requirements.yaml:24:    version: 1.1.7 
istio/requirements.yaml:27:    version: 1.1.7 
istio/requirements.yaml:30:    version: 1.1.7 
istio/requirements.yaml:33:    version: 1.1.7 
istio/requirements.yaml:36:    version: 1.1.7 
istio/requirements.yaml:39:    version: 1.1.7 
istio/requirements.yaml:42:    version: 1.1.7 
istio/charts/certmanager/Chart.yaml:6:version: 1.1.7 
istio/charts/grafana/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/grafana/Chart.yaml:6:version: 1.1.7 
istio/charts/mixer/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/mixer/Chart.yaml:13:version: 1.1.7 
istio/charts/tracing/Chart.yaml:6:version: 1.1.7 
istio/charts/galley/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/galley/Chart.yaml:13:version: 1.1.7 
istio/charts/sidecarInjectorWebhook/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/sidecarInjectorWebhook/Chart.yaml:13:version: 1.1.7 
istio/charts/kiali/Chart.yaml:7:version: 1.1.7 
istio/charts/security/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/security/Chart.yaml:13:version: 1.1.7 
istio/charts/istiocoredns/values.yaml:6:coreDNSImage: coredns/coredns:1.1.7 
istio/charts/istiocoredns/Chart.yaml:6:version: 1.1.7 
istio/charts/gateways/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/gateways/Chart.yaml:15:version: 1.1.7 
istio/charts/prometheus/Chart.yaml:6:version: 1.1.7 
istio/charts/pilot/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/pilot/Chart.yaml:13:version: 1.1.7 
istio/charts/nodeagent/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/nodeagent/Chart.yaml:13:version: 1.1.7 
istio/charts/servicegraph/Chart.yaml:2:appVersion: 1.1.7 
istio/charts/servicegraph/Chart.yaml:6:version: 1.1.7 
istio/README.md:110:    $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/1.1.7 /charts
istio/values.yaml:114:  tag: 1.1.7 
istio/Chart.yaml:2:appVersion: 1.1.7 
istio/Chart.yaml:17:version: 1.1.7 

Any suggestions?

Comment (howardjohn) [reply release/gcb/helm_values.sh]: Could be explicit and replace only tag, appVersion, or version maybe?

Comment (utka) [reply release/gcb/helm_values.sh]: Done

**Comment by howardjohn** [comment]
@utka @duderino we need this in release-1.1 branch if we do another 1.1 release

**Pull request merged**